src/app/containers/Include/index.jsx

```diff
@@ -18,13 +20,35 @@ const IncludeContainer = ({ html, type }) => {
     return null;
   }
 
+  const requireIncludeTypes = ['vj', 'idt1'];
+
+  const paths = `{
+    'jquery-1':
+      'https://static.bbc.co.uk/frameworks/jquery/0.4.1/sharedmodules/jquery-1.7.2',
+    'istats-1':
+      'https://nav.files.bbci.co.uk/orbit-webmodules/0.0.2-448.45dcd56/istats/istats-1',
```

Comment:
This asset is maintained by the Search and Navigation team; the PAL uses Orbit so can rely on Orbit to provide it. We should probably host this ourselves or find an existing version we host.

For context, this a decomissioned analytics library that we only need to include to prevent issues in old includes.

Reply:
In the email about removal of Forge dependencies it says: iStats will be removed from Orbit. Will that be an issue? Does hosting it ourselves solve the issue (if it is one)?

Reply:
That is precisely the issue we'll need to address I think, if anyone knows if this is hosted somewhere in the news estate, it would be good to switch this out.

Reply:
I have spoken with @eagerterrier who is going to host the istats library somewhere for us to use, we can swap this out when this is available.

Reply:
@eagerterrier has shared this asset we can use: https://news.files.bbci.co.uk/include/vjassets/js/vendor/istats-1.0.0.min.js

Reply:
Applied this asset here: https://github.com/bbc/simorgh/pull/6347/commits/7d702752e30ffa76f42f86aef1e061a3feff79a7